packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -363,6 +363,25 @@ export const gliner = (model: ModelData): string[] => [
 model = GLiNER.from_pretrained("${model.id}")`,
 ];
 
+
+export const htrflow = (model: ModelData): string[] => [
+	`# CLI usage
+# see docs: https://ai-riksarkivet.github.io/htrflow/latest/getting_started/quick_start.html
+htrflow pipeline <path/to/pipeline.yaml> <path/to/image>`
+        , `# Python usage
```

Comment:
```suggestion
# see docs: https://ai-riksarkivet.github.io/htrflow/latest/getting_started/quick_start.html
htrflow pipeline <path/to/pipeline.yaml> <path/to/image>`,
        `# Python usage
```

I think this is what's needed to keep 2 snippets while fixing the lint CI

Reply:
That's better than my suggestion!